includes/class-apple-news.php

```diff
@@ -79,6 +79,32 @@ public static function get_filename( $path ) {
 	}
 
 	/**
+	 * Validate settings and see if any updates need to be performed.
+	 *
+	 * @param array $wp_settings An array of settings loaded from WP options.
+	 *
+	 * @access public
+	 * @return array The modified settings array.
+	 */
+	public function validate_settings( $wp_settings ) {
```

Comment:
Not sure if I am reading the comments below correctly, so I thought I'd ask: Does this param need to be optional to accommodate for older versions?